src/components/common/displayfee/DisplayFee.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Chip, { ChipProps } from '../chip/Chip';
+import CopyButton from '../copy_button/CopyButton';
+import { getFee } from '../utils';
+
+export default function DisplayFee({ item, network }: any) {
+    return (
+        <div>
+            <div className="flex items-center text-rgiht gap-2">
+                <span>{getFee(item, network).value}</span>
```

Comment:
can you use a state for the return value instead of calling the function twice ?

`const [fee, setFee] = useState(getFee(item, network));`

you can use fee.value and fee.gas.children after that